source/_posts/Video-js-5-11-0-Prelease.md

```diff
@@ -0,0 +1,104 @@
+---
+title: Video.js 5.11.0 Prelease
+author:
+  name: Gary Katsevman
+  github: gkatsev
+date: 2016-07-22 15:46:51
+tags:
+   - news
+   - releases
+   - version
+---
+Today sees the prerelease of version 5.11.0. I wanted to take a moment to talk about some of the changes, additions, and known issues.
+
+This is a pre-release only. It's available on npm under the `next` tag and also available on the CDN under the fully qualified version number: `//vjs.zencdn.net/5.11.0/video.js`. It'll stay in pre-release state for around a week or more to make sure that there aren't any major glaring bugs, so, please give it a shot and [open issues](https://github.com/videojs/video.js/issues/new) if you find anything.
+
+## Notable Changes
+* In version 5.0, we wanted to deprecate `videojs.players` property in favor of of the `videojs.getPlayers()` getter. However, it's proved to be very useful and it is now being un-deprecated so it will no longer print deprecations in the console.
+* If the player is created without a source set and a user hits play, video.js will now wait for the a source to be provided before starting playback. This eliminates an error that happens when we try to play an empty source.
+* Our custom captions settings dialog was updated to be more accessible by using more aria attributes and better option names in drop downs.
+
+## Known Issues
+* In video.js 5.10, to be able to better respond to the source of the video element changing directly without going through video.js, we [disposed SourceHandlers on subsequent loadstarts](https://github.com/videojs/video.js/pull/3285) and [cleared out the current source](https://github.com/videojs/video.js/pull/3314). However, this causes an [issue with videojs-contrib-dash](https://github.com/videojs/video.js/issues/3428). A contributor investigated and found out that the two PRs mentioned before cause the issue. If a video is created with an MPEG-DASH source element, we end up seeing a `loadstart` event because of the source element and then when Dash.js kicks in and starts playback using MSE, we get another `loadstart` event. Since we see the second `loadstat` event we dispose of the SourceHandler, that is, videojs-contrib-dash and Dash.js, and the video doesn't play. A work around right now while we figure out a correct solution would be not to use source elements in with DASH sources and only use the video.js API in the meantime.
```

Comment:
Seems like there might be a word missing in "... not to use source elements in [HTML?] with DASH sources ...", or am I misreading this?